kubetest/local.go

```diff
@@ -70,10 +71,27 @@ func (n localCluster) Up() error {
 	if err != nil {
 		return err
 	}
+
 	cmd := exec.Command(script)
 	cmd.Env = os.Environ()
 	cmd.Env = append(cmd.Env, "ENABLE_DAEMON=true")
 	cmd.Env = append(cmd.Env, fmt.Sprintf("LOG_DIR=%s", n.tempDir))
+
+	// when we are running in a DIND scenario, we should use the ip address of
+	// the docker0 network interface, This ensures that when the pods come up
+	// the health checks (for example for kubedns) succeed. If there is no
+	// docker0, just use the defaults in local-up-cluster.sh
+	getIp := `ip route list | grep docker0 | awk '{print $9}'`
```

Comment:
I think you can do it in go by iterating `net.Interfaces()`?

like https://stackoverflow.com/questions/23558425/how-do-i-get-the-local-ip-address-in-go?

*bash is evil...*

Reply:
yep, done! thanks for the nudge :)

Reply:
(forgot to push?)

Reply:
whoops!. pushed